### DISCUSSION TITLE
A Jupyter Book square logo idea

### DISCUSSION THREAD

**Discussion by choldgraf** [executablebooks/meta#368] [Ideas]
Had an idea recently for the Jupyter Book logo, just wanted to share below. It involves rotating the curly bracket so that it looks like a book, I quite like it!

![image](https://user-images.githubusercontent.com/1839645/119242334-d823e000-bb11-11eb-9042-5ad914dcd0d5.png)

**Comment by jstac** [comment]
Very nice!  It might be good to add it somewhere on the landing page https://jupyterbook.org/intro.html so it becomes easily recognizable.

**Reply by choldgraf** [reply]
here's how it looks with a few other sub-projects. I also tried using upper-case so that the letters are always aligned with one another:

![image](https://user-images.githubusercontent.com/1839645/119271382-225e9d00-bbb6-11eb-81a1-0ae5f9d24e16.png)

**Reply by choldgraf** [reply]
Here's one with some slight variations on color to emphasize the "book", and including a dark theme

![image](https://user-images.githubusercontent.com/1839645/119270132-1374ec00-bbb0-11eb-9a68-4b09a16a529f.png)

**Reply by damianavila** [reply]
>  It involves rotating the curly bracket so that it looks like a book

Ja, now I see it, nice!!
